TASVideos/Pages/Shared/Components/GameName/Default.cshtml

```diff
@@ -1,8 +1,6 @@
 ﻿@model IList<GameNameModel>
 @if (Model.Any(g => g.IsSystem))
-{
```

Comment:
The "proper" way to fix issues like this in razor is to use the <text> tag:
@if (foo)
{
  <text>@SomeString</text>
}

Reply:
What?

Reply:
https://stackoverflow.com/questions/4969065/how-to-use-text-tag-in-mvc-3-razor